book_config.py

```diff
@@ -130,75 +111,43 @@ def __init__(self, db, ids, parent):
     @property
     def book(self):
         return self._book_settings[self._index]
-    
-    def edit_asin(self, val):
-        self.book.asin = val
 
-    def edit_shelfari_url(self, val):
-        http_string = 'http://www.'
+    def edit_goodreads_url(self, val):
+        goodreads_string = 'https://www.goodreads.com/'
         index = 0
-        if val[:len(http_string)] != http_string:
+        if val[:len(goodreads_string)] != goodreads_string:
             for i, letter in enumerate(val):
-                if i < len(http_string):
-                    if letter == http_string[i]:
+                if i < len(goodreads_string):
+                    if letter == goodreads_string[i]:
                             index += 1
                     else:
                         break
                 else:
                     break
-            self.shelfari_url_edit.setText(http_string + val[index:])
-
-        self.book.shelfari_url = val
-
-    def search_for_asin(self):
```

Comment:
see discussion elsewhere: should we be keeping ASIN code, so we can do a lookup using it?  Also, potentially adding ISBN lookups..